train.py

```diff
@@ -1,15 +1,31 @@
-import argbind
+import time
 from datetime import datetime
 from pathlib import Path
-import time
-from torch.utils.data import DataLoader
-import torch.optim as optim
+
+import argbind
 import torch
+import torch.optim as optim
+from torch.utils.data import DataLoader
 from torch.utils.tensorboard import SummaryWriter
+from tqdm import tqdm
 
 import data
-from model import Scribe
 import utils
+from model import Scribe
+
+
+@argbind.bind
+def scribe_loss(prediction, target, target_mask, penup_weighting=.5):
+    mse_loss = torch.nn.MSELoss(reduction='none')
```

Comment:
Are we instantiating this at each training step?

Reply:
Fixed to instantiate only once.